examples/chart/teleport-cluster/templates/pdb.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
It is normally frowned upon to set the namespace in chart resources but I have done this as it is already done everywhere else.